chia/daemon/server.py

```diff
@@ -1181,15 +1176,15 @@ def daemon_launch_lock_path(root_path: Path) -> Path:
     A path to a file that is lock when a daemon is launching but not yet started.
     This prevents multiple instances from launching.
     """
-    return root_path / "run" / "start-daemon.launching"
+    return service_launch_lock_path(root_path, "daemon")
 
 
 def service_launch_lock_path(root_path: Path, service: str) -> Path:
     """
-    A path to a file that is lock when a service is running.
+    A path that is locked when a service is running.
     """
     service_name = service.replace(" ", "-").replace("/", "-")
-    return root_path / "run" / f"{service_name}.lock"
+    return Path(root_path / "run" / service_name)
```

Comment:
passing this to the `Path()` constructor here looks redundant. `root_path` is already a `Path` object, right? that's why `/` works as path concatenation. Presumably the resulting value from `/` is also a `Path`, right?